src/objective-c/GRPCClient/private/GRPCConnectivityMonitor.m

```diff
@@ -58,45 +58,50 @@ - (BOOL)isCell { \
 }
 
  */
-#define GRPC_XMACRO_ITEM(methodName, FlagName) \
-- (BOOL)methodName { \
-  return !!(_flags & kSCNetworkReachabilityFlags ## FlagName); \
-}
+#define GRPC_XMACRO_ITEM(methodName, FlagName)                 \
+  -(BOOL)methodName {                                          \
+    return !!(_flags & kSCNetworkReachabilityFlags##FlagName); \
+  }
 #include "GRPCReachabilityFlagNames.xmacro.h"
 #undef GRPC_XMACRO_ITEM
 
 - (BOOL)isHostReachable {
-  // Note: connectionOnDemand means it'll be reachable only if using the CFSocketStream API or APIs
+  // Note: connectionOnDemand means it'll be reachable only if using the
+  // CFSocketStream API or APIs
   // on top of it.
-  // connectionRequired means we can't tell until a connection is attempted (e.g. for VPN on
+  // connectionRequired means we can't tell until a connection is attempted
+  // (e.g. for VPN on
   // demand).
-  return self.reachable && !self.interventionRequired && !self.connectionOnDemand;
+  return self.reachable && !self.interventionRequired &&
+         !self.connectionOnDemand;
 }
 
 - (NSString *)description {
   NSMutableArray *activeOptions = [NSMutableArray arrayWithCapacity:9];
 
-  /*
-   * For each flag, add its name to the array if it's ON. Example:
+/*
+ * For each flag, add its name to the array if it's ON. Example:
 
-  if (self.isCell) {
-    [activeOptions addObject:@"isCell"];
-  }
+if (self.isCell) {
+  [activeOptions addObject:@"isCell"];
+}
 
-   */
+ */
```

Comment:
If clang-format is going to align this comment with the following `#define`, then both things should be indented, being part of the `description` method. Do you want to separate these things in a different PR? They seem to require plenty of review, and are irrelevant to the feature which is kind of urgent IIUC.